scss/adminLessonCard.module.scss

```diff
@@ -0,0 +1,43 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 2% 3% 2% 3%;
```

Comment:
Why percent rather than `px`? It's unusual to set the padding or margin as a percentage.

Reply:
I was using % thinking the padding would dynamically adjust on the page depending on how large the overall component will be. I initially thought using a fixed px might make it not as scalable when taking into account the eventual adjustment in size to fit into the lessons home page. I can change to px if this is best practice.

Reply:
`px` for padding is best practice, you want the padding to always be the same. Usually the only thing that changes with screen size is:

1. the text should scale with screen size
2. Overall layout of the page should change with screen size